doc/deparam.md

```diff
@@ -0,0 +1,67 @@
+@function can-route.deparam deparam
+@parent can-route.static
+
+@description Extract data from a route path.
+
+@signature `route.deparam(url)`
+
+  Extract data from a url, creating an object representing its values.
+
+  ```js
+  import {route} from "can";
+
+  const result = route.deparam("page=home");
+  console.log( result.page ); //-> "home"
+  ```
+  @codepen
+
+  @param {String} url A route fragment to extract data from.
+  @return {Object} An object containing the extracted data.
+
+@body
+
+Creates a data object based on the query string passed into it. This is useful to create an object based on the `location.hash`.
```

Comment:
Should this be under `## Use`?

Also, it just starts with "creates" like it is in the description.  Once in the @body, things should sound a bit more 'personal'.